packages/hash/external-services/docker-compose.yml

```diff
@@ -120,7 +120,7 @@ services:
       HASH_GRAPH_PG_PORT: "5432"
       HASH_GRAPH_PG_DATABASE: "${HASH_GRAPH_PG_DEV_DATABASE}"
       HASH_GRAPH_LOG_FORMAT: "${HASH_GRAPH_LOG_FORMAT:-pretty}"
-      HASH_GRAPH_OTLP_ENDPOINT: "http://telemetry-collector:4317"
+#      HASH_GRAPH_OTLP_ENDPOINT: "http://telemetry-collector:4317"
```

Comment:
This is a drive-by. The bug we discovered earlier meant that when I had an invalid domain URL we just didn't get any errors, the Graph deadlocked itself

Reply:
This is tracked in https://app.asana.com/0/1201095311341924/1203636955054323/f